src/client/features/search/gene-results-view.js

```diff
@@ -0,0 +1,113 @@
+const React = require('react');
+const h = require('react-hyperscript');
+const { Link } = require('react-router-dom');
+const queryString = require('query-string');
+const _ = require('lodash');
+
+const { MAX_SIF_NODES, NS_HGNC_SYMBOL, NS_GENECARDS, NS_NCBI_GENE, NS_UNIPROT } = require('../../../config');
+
+const SUPPORTED_COLLECTIONS = new Map([
+  [NS_GENECARDS, 'GeneCards'],
+  [NS_HGNC_SYMBOL, 'HGNC'],
+  [NS_NCBI_GENE, 'NCBI Gene'],
+  [NS_UNIPROT, 'UniProt']
+]);
+
+const getHgncFromXref = xrefLinks => {
+  let symbol;
+  const hgncXrefLink = _.find( xrefLinks, link  => link.namespace === NS_HGNC_SYMBOL );
+  if( hgncXrefLink ) symbol = _.last( _.compact( hgncXrefLink.uri.split('/') ) );
+  return symbol;
+};
+
+class EntitySummaryBox extends React.Component {
+  render(){
+    let { summary } = this.props;
+    let { displayName, xrefLinks } = summary;
+    const hgncSymbol = getHgncFromXref( xrefLinks );
```

Comment:
`props` will now contain values for "geneSymbol" so we can ditch `getHgncFromXref`

Reply:
I dont see geneSymbol in the props object.  Can you be more specific?

Reply:
These are the only things I see in the props object.  I don't see geneSymbol anywhere.
![screen shot 2018-12-10 at 11 44 24 am](https://user-images.githubusercontent.com/2328291/49747248-00386500-fc71-11e8-991c-e3d57afcea7e.png)

Reply:
Nevermind, I found it.